CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 Changelog
 =========
+# 0.49.3-SNAPSHOT / 2024-06-07
+
```

Comment:
```suggestion
* [IMPROVEMENT] Updated version to jmxfetch-0.49.3-SNAPSHOT
```

Not sure if `IMPROVEMENT` or something else, or if there can be a better way to word this, but should probably have some text here to note what the change was.

Reply:
This is just a placeholder for the next release so will be updated when we are ready to create a new release.